src/main/scala/kamon/jmx/collector/Configuration.scala

```diff
@@ -30,7 +30,7 @@ private[collector] trait Configuration {
       val metricName: String = nameObj.unwrapped().asInstanceOf[String]
       val jmxQuery: String = queryObj.unwrapped().asInstanceOf[String]
 
-      val attributesList: Seq[(String, Any)] = for {
+      val attributesList: collection.Seq[(String, Any)] = for {
```

Comment:
Probably this is not needed, it can be what it was before?

Reply:
This is necessary so the code cross compiles in all versions of Scala...